CMakeLists.txt

```diff
@@ -40,6 +37,8 @@ else()
   find_package(RtMidi CONFIG REQUIRED)
   find_package(RtAudio CONFIG REQUIRED)
 endif()
+find_package(clap CONFIG REQUIRED)
+find_package(clap-helpers CONFIG REQUIRED)
```

Comment:
just wanted to get all the compiler flag stuff before all the `find_package` stuff